components/locale/src/fallback/algorithms.rs

```diff
@@ -256,54 +363,79 @@ mod tests {
                 "sr-Latn-fonipa",
                 "sr-Latn",
             ],
+            expected_script_chain: &[
+                "sr-ME-fonipa",
+                "sr-ME",
+                "sr-fonipa",
+                "sr",
+                "und-Latn-fonipa",
+                "und-Latn",
+                "und-fonipa",
+            ],
```

Comment:
Issue: this is not right because you're switching back and forth between Latin and Cyrillic. Maybe we want that but it should be an explicit parent, I think.

I lean toward

1. `sr-ME-fonipa`
2. `sr-ME`
3. `sr-Latn-ME-fonipa`
4. `sr-Latn-ME`
5. `sr-Latn-fonipa` (?)
6. `sr-Latn` (note: up to here, identical to language fallback)
7. `sr-fonipa` (?*)
8. `sr` (*)
9. `und-Latn-ME-fonipa`
10. `und-Latn-ME`
11. `und-Latn-fonipa` (?)
12. `und-Latn`
13. `und-fonipa` (?)
14. `und`

\* only if `sr` is an explicit parent of `sr-Latn`

Reply:
Issue: `sr-Latn` definitely should be in the chain.

Reply:
It now is